apps/users/models.py

```diff
@@ -227,6 +193,8 @@ class UserProfile(amo.models.OnChangeMixin, amo.models.ModelBase,
     lang = models.CharField(max_length=5, null=True, blank=True,
                             default=settings.LANGUAGE_CODE)
 
+    t_shirt_requested = models.DateTimeField(null=True)
```

Comment:
it's missing `blank=True` (or `editable=False`, but then it won't be editabled in the admin)